### ISSUE TITLE
Accessibility Violation in Common Properties disabled Add Columns text

### ISSUE THREAD

**Issue by srikant-ch5** [elyra-ai/canvas#2492]
Ref: 4042
<img width="1726" alt="Image" src="https://github.com/user-attachments/assets/21f737b3-3be7-4baf-93de-9e8f8a9f5e74" />
